src/frontend/src/lib/utils/tokens.utils.ts

```diff
@@ -152,6 +152,33 @@ export const pinTokensWithBalanceAtTop = ({
 export const sumTokensUiUsdBalance = (tokens: TokenUi[]): number =>
 	tokens.reduce((acc, token) => acc + (token.usdBalance ?? 0), 0);
 
+/**
+ * Calculates total USD balance of mainnet tokens from the provided tokens list.
+ *
+ * @param $tokens - The list of tokens for filtering by network env and total USD balance calculation.
+ * @param $balancesStore - The balances data for the tokens.
+ * @param $exchanges - The exchange rates data for the tokens.
+ * @returns The sum of mainnet tokens USD balance.
+ *
+ */
+export const sumMainnetTokensUsdBalance = ({
```

Comment:
1. Can you type the return type

2. Is it correct to default to zero as result if there is no mainnet token? To some extend how to you know if the sum is 0 because there is a zero balance on mainnet or because there is no tokens related to mainnet?

Reply:
1. Sure.
2. ICP and ETH mainnet tokens are always on by default. If we assume that no-mainnet-tokens case can indeed happen, for me it becomes a product/design question - do we still display "$0" in NetworkSwitcher, do we avoid showing description completely, do we show a small text message with a hint that no tokens available, etc. Do you think we need to be prepared for this case now or is it okay to handle it if it is decided to remove pre-selected tokens feature?